lib/money/simple_money_parser.rb

```diff
@@ -0,0 +1,23 @@
+# Parses amount with optionally a space and currency code. Raises an error if
```

Comment:
I guess would be nice to have a better visualization what "optionally a space and currency code" means.
something like... all of this you wrote + actual example
```ruby
# "USD 10.00"
```